Rewrite/tests/Rewrite.CSharp.Tests/Tree/LiteralTests.cs

```diff
@@ -64,4 +66,27 @@ public class T
             )
         );
     }
+
+    [Fact]
+    private void Null()
+    {
+        RewriteRun(
+            CSharp(
+                """
+                public class T
+                {
+                    object _o = null;
+                }
+                """,
+                spec: spec => spec.AfterRecipe = cu =>
+                {
+                    var o = (J.VariableDeclarations)((J.ClassDeclaration)cu.Members[0]).Body.Statements[0];
+                    o.Should().NotBeNull();
+                    var nullLiteral = (J.Literal)o.Variables[0].Initializer!;
```

Comment:
@macsux What you are saying is correct, but as I explained I only do the cast here, so that the `Type` property I reference on the next line will be resolved against `J.Literal.Type` and not `Expression.Type`, because the latter will in C# not do a polymorphic dispatch because of how default interface methods work in C#. In Java I can define a default method on any interface and it will always do a polymorphic dispatch to an override in an implementing class. Do you see what I mean?